helpers/collection/paginate.go

```diff
@@ -0,0 +1,108 @@
+package collection
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/go-pg/pg/orm"
+)
+
+// Paginate is part of filter structure
+type Paginate struct {
```

Comment:
if you wanna to do all of this options, it must have name like `Form`...

Reply:
to ```Form```